### PR TITLE
DBZ-2565 Process infinite timestamps

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-2565

### CODE REVIEW

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java [hunk 1]
```diff
@@ -608,7 +608,8 @@ private long asEpochMicros(String timestamp) {
                 new SchemaAndValueField("ts_large", MicroTimestamp.builder().optional().build(), expectedTsLarge),
                 new SchemaAndValueField("ts_large_us", MicroTimestamp.builder().optional().build(), expectedTsLargeUs),
                 new SchemaAndValueField("ts_large_ms", Timestamp.builder().optional().build(), expectedTsLargeMs),
-                new SchemaAndValueField("tz_large", ZonedTimestamp.builder().optional().build(), expectedTzLarge));
+                new SchemaAndValueField("tz_large", ZonedTimestamp.builder().optional().build(), expectedTzLarge),
+                new SchemaAndValueField("ts_pinf", MicroTimestamp.builder().optional().build(), Long.MAX_VALUE));
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java:612]: Where's the values for ts_minf and tz_minf?

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java [hunk 2]
```diff
@@ -108,6 +110,16 @@
...
 
+    /**
+     * A value used by Debezium to define timestamp as positive infinity.
+     */
+    public static final Instant TIMESTAMP_POSITIVE_INFINITY = Conversions.toInstantFromMicros(Long.MAX_VALUE);
+
+    /**
+     * A number used by Debezium to define timestamp as negative infinity.
+     */
+    public static final Instant TIMESTAMP_NEGATIVE_INFINITY = Conversions.toInstantFromMicros(Long.MAX_VALUE + 1);
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java:121]: Why not Long.MIN_VALUE?

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java (hunk 1 above)
Comment (gunnarmorling) [reply debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java:612]: Btw. how does this PR build not fail for decoderbufs, without the plug-in PR applied 🤔 ?

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java (hunk 2 above)
Comment (jpechane) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java:121]: Good question :-)

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java (hunk 1 above)
Comment (jpechane) [reply debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java:612]: It is easy, the CI build runs against `9.6-devel` tag that has the change applied

**Comment by jpechane** [comment]
This PR should serve as a basis for https://issues.redhat.com/browse/DBZ-2614

**Comment by jpechane** [comment]
Replaced with https://github.com/debezium/debezium/pull/1879

**Pull request closed without merging**